.gitignore

```diff
@@ -1,8 +1,21 @@
-.DS_Store
-
 # Xcode
 #
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
```

Comment:
Nitpick. What do you think of this?

```suggestion
# Taken from
# https://raw.githubusercontent.com/github/gitignore/991e760c1c6d50fdda246e0178b9c58b06770b90/Swift.gitignore

# Xcode
```

Mainly, I find it confusing to read mentions to the other `.gitignore` files that are part of the [gitgnore](https://github.com/github/gitignore/) repo but not of this.

Reply:
Oh ya this is super nice!!